cli/ballerina-cli/src/main/java/io/ballerina/cli/task/CreateExecutableTask.java

```diff
@@ -137,6 +141,12 @@ public void execute(Project project) {
             }
         }
 
+        for (Diagnostic d : jarResolverDiagnostics) {
+            if (d.diagnosticInfo().severity().equals(DiagnosticSeverity.WARNING)) {
```

Comment:
Do we need to check for Warnings? We can just print the diagnostics IMO

Reply:
Fixed with 427054e